pkg/apis/openebs.io/upgrade/v1alpha1/upgraderesult.go

```diff
@@ -36,17 +36,22 @@ type UpgradeResult struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Config UpgradeResultConfig `json:"config"`
+	Config UpgradeConfig `json:"config"`
 	// Tasks are the runtasks that needs to be
 	// executed to perform this upgrade
 	Tasks  []UpgradeResultTask `json:"tasks"`
 	Status UpgradeResultStatus `json:"status"`
 }
 
-// UpgradeResultConfig represents the entire config
-// of UpgradeResult i.e. same as task-executor job config
-type UpgradeResultConfig struct {
-	// TODO: Add Task-executor job config here
+// UpgradeConfig represents the config of UpgradeResult i.e.
+// It contains resource details of single unit of upgrade and
+// all runtime configuration.
+type UpgradeConfig struct {
```

Comment:
Can we rename this to `UpgradeResultConfig`

It will be in the same lines as `UpgradeResultTask` & `UpgradeResultStatus`

Reply:
Changed. I thought other way that's why I changed both config names.